src/app/utils/useDocumentTitle.ts

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+// a custom hook for setting the page title
+function useDocumentTitle(title: string) {
+  React.useEffect(
+    () => {
+      const originalTitle = document.title;
+      document.title = title;
+
+      return () => {
+        document.title = originalTitle;
+      };
+    },
+    [title]
+  );
+}
+
+export { useDocumentTitle };
```

Comment:
Why not exporting the function directly? `export function useDocumentTitle(...`

Reply:
No particular reason 😀 At first I was thinking there may be multiple exports, but now I realize that's probably not the case. Will simplify it 👍

Reply:
Even if there are multiple exports, the `export` keyword will work

```
export function foo() {}
export function bar() {}
export function baz() {}
```

is equivalent to

```
function foo() {}
function bar() {}
function baz() {}
export { foo, bar, baz };
```

Reply:
Yerp, realized that right after I posted the comment 🤦‍♂️